### DISCUSSION TITLE
Hi, I'm new, how do I monitor windows? (I don't want to use netdata cloud)

### DISCUSSION THREAD

**Discussion by Klewltz** [netdata/netdata#13832] [Q&A]
I want to monitor Windows (I know the windows exporter), but I don't know how to do it

**Comment by MrZammler** [comment]
Hi @LiSiTuS10 !

Have you check the relevant part of the documentation https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/wmi/ ?
